doc/api/module.md

```diff
@@ -76,6 +76,14 @@ const requireUtil = createRequireFromPath('../src/utils/');
 requireUtil('./some-tool');
 ```
 
+### `module.isPreloading`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {boolean} True if the module is running during the Node.js preload
```

Comment:
Maybe it is worth explicitly clarifying this has no bearing on ESM?

Reply:
I'll open a separate pr that clarifies that preload modules only support cjs.